### ISSUE TITLE
Add CLI option to use different build profiles

### ISSUE THREAD

**Issue by volks73** [volks73/cargo-wix#91]
As mentioned in #90, the cargo-wix subcommand builds the project every time before the compiling and linking steps of the MSI using the Release build profile. This is currently hard-coded, but a flag or option should be added to allow changing the build profile.

If and when #20 is completed, this should also be added as a field.

**Comment by volks73** [comment]
From #93, the correct terminology is "release profiles", not "build profiles", from the [Cargo book](https://doc.rust-lang.org/book/ch14-01-release-profiles.html).

This enhancement is to add a CLI flag or option to use `cargo build --debug` instead of `cargo build --release`. Currently, `cargo build --release` is hard-coded into the `create`/default subcommand.

**Comment by volks73** [comment]
This has been implemented as of e21030ff7e71c4dd143eb6f48301c1dcedad940b.

The implementation includes two new CLI flags and meatadata fields for the default _create_ subcommand: `-d,--dbg-build` and `-D,--dbg-name`. The `-d,--dbg-build` flag will build the package using the Debug profile while the `-D,--dbg-name` flag will append `-debug` to the file stem (string before the file extension) of the installer's file name. These can be used in combination to indicate to end-users the installer has a binary built with debugging information. They are mutually exclusive, so it is possible to use one without the other. For example, if there is some other mechanism for debugging an application and a debug and release variant are desired, only the `-D,--dbg-name` flag is needed.

The `--dbg-build` and `--dbg-name` long names are used to avoid confusion with "debug" of the cargo-wix subcommand. I did not want to use `--debug` or `--dbg` because it was not clear what was happening with the debug. I believe `--dbg-build` is more descriptive. The metadata fields are similarly `dbg-build` and `dbg-name` and they are boolean values.

Tests have been added and pass.
